worker-websocket-durable-objects/tsconfig.json

```diff
@@ -0,0 +1,105 @@
+{
+  "compilerOptions": {
+    /* Visit https://aka.ms/tsconfig.json to read more about this file */
+
+    /* Projects */
+    // "incremental": true,                              /* Enable incremental compilation */
```

Comment:
Can we remove all the comments? It should be clean & just contain relevant/useful configuration IMO. We can/should also use the basic `tsconfig.json` from the `worker-typescript` example